### DISCUSSION TITLE
[27 giugno 2023] Esercizio 5

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/psd2223#256] [Esami]
Data l’espressione booleana $\Large f$, semplificarla usando gli assiomi ed I teoremi dell’algebra di Boole e scrivere la forma POS minima.

<div align="center">

![image](https://github.com/user-attachments/assets/88bac3bc-4ea7-460c-8b79-b80632434167)

</div>

**Comment by M1SC0** [comment]
![image](https://github.com/user-attachments/assets/23e5dd44-a9f6-4a1b-8357-dc75cbfd7ddc)
